### PR TITLE
fix: stop automatic compensation for pitch on camera head tilt

### PR DESCRIPTION
**Pull request by jhonasiv**
Depends on:
- [ ] https://github.com/tidewise/bundles-rov/pull/119

### CODE REVIEW

File: src/CommandAndStateMessageParser.cpp [hunk 1]
```diff
@@ -611,14 +614,25 @@ RigidBodyState CommandAndStateMessageParser::getCameraHeadTiltMotorStateRBS(
 
     RigidBodyState rbs;
     rbs.time = Time::now();
-    auto yaw = root["tilt"]["position"].asDouble() * M_PI / 180;
+    auto tilt = root["tilt"]["position"].asDouble() * M_PI / 180;
```
Comment (doudou) [inline src/CommandAndStateMessageParser.cpp:617]: camera_head2world_tilt

File: src/CommandAndStateMessageParser.cpp [hunk 2]
```diff
@@ -597,8 +597,11 @@ samples::Joints CommandAndStateMessageParser::getCameraHeadTiltMotorState(string
 
     samples::Joints motor_states;
     motor_states.time = Time::now();
+    auto tilt = root["tilt"]["position"].asDouble() * M_PI / 180;
```
Comment (doudou) [inline src/CommandAndStateMessageParser.cpp:600]: camera_head2world_tilt

And use base::Angle in the calculations to make sure the end position is normalized (given that the head can move almost 180, that's definitely needed)

File: src/CommandAndStateMessageParser.cpp (hunk 1 above)
Comment (jhonasiv) [reply src/CommandAndStateMessageParser.cpp:617]: Do you mean passing the JointState in the caller as an argument to the getCameraHead* methods? If so, the caller is the Task, it doesnt have access to the JSON, which is a private member. For the second option, I think that's better as well.

**Comment by jhonasiv** [comment]
Ping @kapeps @doudou @seixasxbr

**Pull request merged**